lib/Eclair/EIR/Lower.hs

```diff
@@ -7,6 +7,7 @@ module Eclair.EIR.Lower
 import Prelude hiding (void)
 import qualified Prelude
 import qualified Relude (swap)
+import Control.Monad.Fix
```

Comment:
Re-export this from the Eclair Prelude, we use it in so many places, we might as well export it there..

Reply:
Since you already made the changes you suggested above, I am resolving this